### DISCUSSION TITLE
Running a play list from command line.

### DISCUSSION THREAD

**Discussion by HamishBrownPFR** [APSIMInitiative/ApsimX#9465] [Q&A] [answered]
It is possible to run a sub-set of the simulations from an .apsimx file from command line by passing a list of simulations in.  The example below uses the subprocess library in python to execute the command line run.

```python
subprocess.run(['C:/GitHubRepos/ApsimX/bin/Debug/net6.0/Models.exe',
               'C:/GitHubRepos/ApsimX/Prototypes/Lentil/Lentil.apsimx',
               '/SimulationNameRegexPattern:(SimName1)|(SimName2)|(SimName3)|(SimName4)'],
               stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```
APSIM now has the Playlist class which can select a sub set of simulations to run which is easier than trying to compile lists of simulations in a script.  However, I cannot work out how to run the playlist from the command line.  
Anyone know how to do that or if it is even possible?

**Comment by ric394** [comment, marked as answer]
Thanks for bringing this up @HamishBrownPFR. This has exposed a gap in our documentation. I'll write up a 'how to' in the command line documentation. This will be available shortly.

**Reply by HamishBrownPFR** [reply]
Thanks @ric394, much appreciated
